backend/src/main/java/com/google/sps/data/MatchPreference.java

```diff
@@ -0,0 +1,44 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+/** Represent whether or not a Participant is matched */
```

Comment:
Is this  new class is meant to replace match status ?

Reply:
Or alongside it , this comment suggests the former

Reply:
MatchStatus (whether or not a participant is matched yet) and MatchPreference (whether they want to be matched with someone similar, different, or anyone) are different.

Reply:
Okay !